maquetado/scss/_tech.scss

```diff
@@ -52,15 +37,12 @@
 }
 
 .tech-list {
-  display: flex;
-  flex-wrap: wrap;
-  padding-left: 100px;
-
-  &:not(:last-child) {
-    margin-bottom: 100px;
-  }
-
-  .tech {
-    margin: 20px;
-  }
+  align-items: center;
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: repeat(3, $tech-width);
+  grid-template-rows: repeat(2, auto);
```

Comment:
I think that this is unnecessary, please delete it

Reply:
Delete what? Line #44?

Reply:
Yes